.repo/sample-list-generator/src/fetch.ts

```diff
@@ -0,0 +1,70 @@
+import fetch from 'node-fetch';
```

Comment:
The file name fetch is a bit misleading as the code seems to do something completely different. Can you find a. better name?